### PR TITLE
Saleor 2960 checkout products without shipment

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for purchase products by type & removed unnecessary functions from utils files. 

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2960 Create tests for product without shipment option](https://app.clickup.com/t/2549495/SALEOR-2960)

**Review by orzechdev — APPROVED** [review]

File: cypress/integration/configuration/productTypes/purchaseWithProductTypes.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+          billingAddress: address,
+          variantsList,
+          auth: "token"
+        });
+      })
+      .then(({ order }) => {
+        getOrder(order.id);
+      })
+      .then(order => {
+        expect(order.status).to.be.eq("UNFULFILLED");
```
Comment (eCzerniak) [inline cypress/integration/configuration/productTypes/purchaseWithProductTypes.js:110]: Can you add assertion to check isShippingRequired to all cases? :)

File: cypress/apiRequests/Checkout.js [hunk 2]
```diff
@@ -33,7 +37,9 @@ export function createCheckout({
       }
     }
   }`;
-  return cy.sendRequestWithQuery(mutation, auth);
+  return cy
+    .sendRequestWithQuery(mutation, auth)
+    .its("body.data.checkoutCreate.checkout");
 }
```
Comment (eCzerniak) [inline cypress/apiRequests/Checkout.js:43]: Please add empty line between functions.

File: cypress/integration/configuration/productTypes/purchaseWithProductTypes.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+  let address;
+  let warehouse;
+  let attribute;
+  let category;
+  let shippingMethod;
+  let createProductData;
+
+  before(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    deleteProductsStartsWith(startsWith);
```
Comment (eCzerniak) [inline cypress/integration/configuration/productTypes/purchaseWithProductTypes.js:44]: Is this lack of indentation is intentional?

Comment (karola312) [reply cypress/integration/configuration/productTypes/purchaseWithProductTypes.js:44]: Yes, there is no need for indentation if line before ended with ;

File: cypress/integration/configuration/productTypes/purchaseWithProductTypes.js (hunk 1 above)
Comment (karola312) [reply cypress/integration/configuration/productTypes/purchaseWithProductTypes.js:110]: I added expect to check isShippingRequired has correct value in order for all cases

**Review by eCzerniak — APPROVED** [review]

**Review by jwm0 — APPROVED** [review]

**Pull request merged**